src/antsibull/data/ansible-community.py.j2

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+# coding: utf-8
+# Author: Mario Lenz <m@riolenz.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2022
+"""The ansible-community CLI program."""
+
+import argparse
+
+
+def main():
+    '''Main entrypoint for the ansible-community CLI program.'''
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--version',
+        action='version',
+        version='Ansible community version {{ version }}',
+        help="show the version of the Ansible community package",
+    )
+    args = parser.parse_args()  # noqa: F841, pylint: disable=unused-variable
```

Comment:
Could not the return value not used at all?

Reply:
Do you suggest changing this to `parser.parse_args()`? I thought a bit about that, but decided to keep the line as-is (with the comments) so that this is easier to extend.

Reply:
@felixfontein I think it would be cleaner to have just `parser.parse_args()`. It would be trivial to change it back to `args = parser.parse_args()` when we need it.

Reply:
> Do you suggest changing this to `parser.parse_args()`? I thought a bit about that, but decided to keep the line as-is (with the comments) so that this is easier to extend.

Yes.

Reply:
Implemented.